File: lib/Models/Property/PropertyAddRequest.dart
class PropertyAddRequest {
  final String name;
  final String? imageURL;
  final int quantity;
  final String shortDescription;
  final String? fullDetail;
  final int categoryID;

  const PropertyAddRequest(
      {required this.name,
      required this.imageURL,
      required this.quantity,
      required this.shortDescription,
      required this.fullDetail,
      required this.categoryID});
}

File: backend/InventoryDemo.DTOs/Requests/PropertyAddRequest.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace InventoryDemo.DTOs.Requests
{
    public class PropertyAddRequest
    {
        [Required(ErrorMessage = "The name of a property can not be empty.")]
        public string Name { get; set; }
        public string? ImageURL { get; set; }
        public int Quantity { get; set; }
        public string? ShortDescription { get; set; }
        public string? FullDetail { get; set; }
        public DateTime CreatedDate { get; set; }
        public int CategoryID { get; set; }
    }
}
